scripts/eurostat/gdp/import_data.py

```diff
@@ -21,15 +21,20 @@
 import json
 import pandas as pd
 from preprocess_data import preprocess_df
+import sys
+
+# Imports country mapping alpha2 country codes to country DCIDs.
+sys.path.insert(1, '../../../util')
+from alpha2_to_dcid import COUNTRY_MAP
```

Comment:
import util.alpha2_to_dcid as alpha2_to_dcid

Reply:
This throws the error 
```
Traceback (most recent call last):
  File "import_data.py", line 29, in <module>
    import util.alpha2_to_dcid as alpha2_to_dcid
ModuleNotFoundError: No module named 'util'
```
This is a common python issue. See https://stackoverflow.com/questions/4383571/importing-files-from-different-folder for details.